client/src/modules/list/index.jsx

```diff
@@ -133,7 +133,7 @@ class List extends Component {
     }
 
     const { cohortId, isArchived, isMember, isOwner, name, type } = list;
-    const isCohortList = cohortId !== null;
+    const isCohortList = cohortId !== null && cohortId !== undefined;
```

Comment:
you can do it just 
`const isCohortList = !cohortId;`

If cohort id will be undefined or null you'll get true. If cohort id will be some data, you will get false.

Reply:
Short example:
![Zrzut ekranu 2019-06-11 o 15 32 42](https://user-images.githubusercontent.com/27632432/59276228-34133800-8c5e-11e9-923f-9fc1ae186605.png)

Reply:
I should do `const isCohortList = cohortId`, but then I receive an  error that cohortID should be a boolean